src/api/index.ts

```diff
@@ -48,7 +48,7 @@ export const snykApiRef: ApiRef<SnykApi> = createApiRef<SnykApi>({
 export interface SnykApi {
   ListAllAggregatedIssues(orgName: string, projectId: string): Promise<any>;
   ProjectDetails(orgName: string, projectId: string): Promise<any>;
-  GetCompleteProjectsListFromAnnotations(orgId: string, annotations: Record<string,string>):Promise<ProjectsData[]>;
+  GetCompleteProjectsListFromAnnotations(orgId: string, annotations: Record<string,string>, ignoreMissingTargets: boolean):Promise<ProjectsData[]>;
```

Comment:
What is the ignoreMissingTargets doing/for? I'm not sure I understand the situation where that'd be needed.

Reply:
The target resolution throws an error if a target slug/id is not found within an organization. However with multiple orgs there will always be a case where a target is not found within an org.
The relevant line is this one here: https://github.com/snyk-tech-services/backstage-plugin-snyk/pull/167/files#diff-764761d79ed791ea154187526b088322dba3bebe94f9d38ab22120554aba77c7R179

And this is where the exception is thrown if a target is not found within an org: https://github.com/Interhyp/backstage-plugin-snyk/blob/1ad9438bec685daa30d9b2d61741f9bea14f4223/src/api/index.ts#L253